stripe/src/main/java/com/stripe/android/paymentsheet/BaseAddCardFragment.kt

```diff
@@ -43,7 +43,6 @@ internal abstract class BaseAddCardFragment(
     private lateinit var cardMultilineWidget: CardMultilineWidget
     private lateinit var billingAddressView: BillingAddressView
     private lateinit var cardErrors: TextView
-    private lateinit var googlePayButton: View
```

Comment:
For custom integration, do we now always show the carousel first, even if there is no customer or the custom doesn't have payment methods?

Reply:
yes, if google pay is enabled and there is no customer, or no payment methods we want to show the carousel